custom/icds_reports/reports/governance_apis.py

```diff
@@ -43,8 +33,9 @@ def base_data(row_data):
             expected_visits=get_value_or_data_not_entered(row_data, 'expected_visits'),
         )
 
+    data_rows = []
     for row in data:
-        config['data'].append(base_data(row))
-    config['data'] = config['data'][int(start):(int(start) + length)]
+        data_rows.append(base_data(row))
+    data_rows = data_rows[int(start):(int(start) + length)]
```

Comment:
help. Catching it now. Applying the pagination here won't buy us performance improvement. this pagination needs to applied on where you are fetching the data

Reply:
yeah.thought so. But saw a few implementations earlier on other reports. so made it like that. will change it